src/components/eventCard/EventCard.js

```diff
@@ -20,19 +20,19 @@ export const EventCard = ({ event }) => {
   return (
     <section key={event.id} className="event-card">
       <dl>
-        <dt>Dog</dt>
-        <dd>{event.dog.name}</dd>
-
-        <dt>Event Name</dt>
-        <dd>
-          <Link to={`/event/${event.id}`} key={event.id} >{event.name}</Link>
-        </dd>
-
-        <dt>Event Date</dt>
-        <dd>{(new Date(event.eventDatetime)).toLocaleString()}</dd>
+        <div className='description-container'>
```

Comment:
what are these divs accomplishing? Could the styling be applied to a different semantic element; i.e. the dl, dt, or dd?

Reply:
For styling, when you use the other tags, it messes it up